.cirrus.yml

```
@@ -397,6 +397,8 @@ asan_sanitizer_task:
     CXXFLAGS: -DZEEK_DICT_DEBUG
     ZEEK_CI_CONFIGURE_FLAGS: *ASAN_SANITIZER_CONFIG
     ASAN_OPTIONS: detect_leaks=1:detect_odr_violation=0
+    # Use absolute paths for coverage files.
```

Could you add a little more context here, e.g.,

```suggestion
    # Use absolute paths for coverage files since gcov gets confused by relative paths.
```